lib/utils.js

```diff
@@ -315,14 +315,14 @@ export function getSiteString(langData = "") {
   if (Array.isArray(langData)) {
     return langData[0] === "es" ? "es" : "default";
   } else {
-    return langData.includes("/es/") || langData === "es" ? "es" : "default";
+    return langData.includes("/es") || langData === "es" ? "es" : "default";
```

Comment:
maybe a solution here would be like:

```
return langData.includes("/es/") || return langData.includes("/es", -3) || langData === "es" ? "es" : "default";
```

that way it first checks for enclosing slashes, then if the last 3 characters of the URL string are `/es`, and then finally the comparison check?